app/build.gradle

```diff
@@ -59,4 +59,5 @@ dependencies {
     implementation 'com.github.bumptech.glide:glide:4.9.0'
     annotationProcessor 'com.github.bumptech.glide:compiler:4.9.0'
 
+
```

Comment:
the whole change to this file is an empty space which triggers unnecessary gradle syncs. please remove 😄